services.yaml

```diff
@@ -4224,6 +4224,148 @@ paths:
           $ref: "#/responses/NotFoundError"
         401:
           $ref: "#/responses/UnauthorizedError"
+#Jira
+  /services/{env_id}/alerts/jira/projects:
```

Comment:
should be `/services/{env_id}/jira/projects`. It has nothing to do with alerts in this context